mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
```

Comment:
The code snippet provided is a part of the `removeOrderById` function in the `order.js` controller. 

Here is the review:

In the `.then(result => {` block, it would be better to check if `result.ok` is true before checking `result.n === 0`. This is because `result.ok` indicates whether the operation was successful, and `result.n` indicates the number of documents affected.